go/master/service.go

```diff
@@ -432,3 +458,46 @@ func (s *Service) TaskFailed(meta TaskMeta, dummy *int) error {
 	s.processFailedTask(t, meta.Epoch)
 	return nil
 }
+
+// ========================== Client Pass Sync Calls ==========================
+
+// AddClient need to be called when a new master client connected.
+// We need to keep a internal client count to sync all clients when pass ends.
+func (s *Service) AddClient(dummyIn int, dummyOut *int) error {
```

Comment:
Sorry for the bad example I created, latter I realized this can be changed to:
```go
func (s *Service) AddClient(_ int, _ *int) error {
  // ...
}
```